src/lib/bridge/handlers/open_options_page.ts

```diff
@@ -0,0 +1,19 @@
+import {RequestHandler} from '../types';
+import {RequestType} from './types';
+import {isFirefox} from '../../utils/detect';
+
+class OpenOptionsPageHandler implements RequestHandler<undefined, void> {
```

Comment:
There are other handlers like `EmptyRequestHandler` that you could use.

I'd propose an `EmptyHandler` implementation in `main.ts` with the following:

```
export class EmptyHandler<Req> implements RequestHandler<Empty, void> {
    constructor(private type: RequestType, private handler: (sender: MessageSender) => Promise<void>) {}

    getType(): RequestType {
        return this.type;
    }

    handleRequest(sender: MessageSender): Promise<void> {
        return this.handler(sender);
    }
}
```

Then you can instantiate this handler instead.